Update Positron Notebook Editor docs: alpha to beta (#346)
posit-dev/positron-docs: positron-notebook-editor.qmd

- **Notebook-aware AI assistance**: AI understands your notebook context, execution history, and execution order
- **Integrated data exploration**: **Variables** pane and Data Explorer work seamlessly
- **Debugging**: Works out of the box without additional setup

> - **Improved version control**: Control what metadata is saved and easily convert Jupyter notebooks to Quarto notebooks for better diffing and version control

Reviewer: Maybe, to suggest converting both ways
```suggestion
- **Improved version control**: Control what metadata is saved and easily convert between Jupyter notebooks (`.ipynb`) and Quarto notebooks (`.qmd`) for better diffing and version control
```

Author: will remove file conversion note until we ship changes

If an environment installed on your computer is not available in Positron, you may want to read more about how Positron discovers [Python installations](python-installations.qmd) and [R installations](r-installations.qmd).

{{< include "_notebook-kernel-selection.qmd" >}}

![The Positron Notebook Editor action bar with the kernel selector highlighted, showing Python 3.12.11 as the selected interpreter.](images/positron-notebook-editor-kernel-selector.png){fig-alt="Notebook Editor action bar with the kernel selector dropdown highlighted in red, showing Python 3.12.11 (Pyenv) selected."}

You can select a different interpreter for the notebook by selecting the **Kernel Selector** in the notebook editor action bar. The **Change Kernel...** option will display a list of all the registered interpreters. Select an interpreter from the list.

Alternatively, you can run the *Positron Notebook: Change Kernel...* command from the Command Palette.

## Version control

> The Positron Notebook Editor controls what metadata is saved to the notebook file. This makes notebooks more version-control-friendly. By default, cell outputs are saved but execution counts are not. You can adjust this behavior with the [`notebook.save.outputs`](positron://settings/notebook.save.outputs) and [`notebook.save.executionCounts`](positron://settings/notebook.save.executionCounts) settings.

Reviewer: ```suggestion
The Positron Notebook Editor controls what metadata is saved to the notebook file. This makes notebooks more version-control-friendly. By default, both cell outputs and execution counts are saved. You can adjust this behavior with the [`notebook.save.outputs`](positron://settings/notebook.save.outputs) and [`notebook.save.executionCounts`](positron://settings/notebook.save.executionCounts) settings.
```

Reviewer: Wdyt about linking to the subsection with these settings below instead of listing individual settings here?

> For cleaner diffs, you can convert a Jupyter notebook to a Quarto notebook (`.qmd`) by running the _Quarto: Convert to Quarto Document_ command from the Command Palette. Quarto notebooks store code and markdown as plain text, making them easier to review in pull requests. For more information on Quarto convert see [the Quarto documentation](https://quarto.org/docs/cli/convert.html).

Author: @seeM is this correct?

> For cleaner diffs, you can convert a Jupyter notebook to a Quarto notebook (`.qmd`) by running the _Quarto: Convert to Quarto Document_ command from the Command Palette. Quarto notebooks store code and markdown as plain text, making them easier to review in pull requests. For more information on Quarto convert see [the Quarto documentation](https://quarto.org/docs/cli/convert.html).

Reviewer: ```suggestion
For cleaner diffs, you can convert a Jupyter notebook to a Quarto notebook (`.qmd`) by running the _Notebook: Convert to .qmd_ command from the Command Palette. Quarto notebooks store code and markdown as plain text, making them easier to review in pull requests. For more information on Quarto convert see [the Quarto documentation](https://quarto.org/docs/cli/convert.html).
```

Selecting an option from the panel will open a new [**Chat** pane](assistant-chat.qmd) with the notebook context already attached.

> ### Ghost cell suggestions (Experimental)

Author: @nstrayer are you happy with this section

Reviewer 2: yup!

Ghost cell suggestions use AI to predict your next step after you execute a cell. A ghost cell appears below the current cell with a suggested action you can accept, edit, or dismiss.

To enable ghost cell suggestions, set [`positron.assistant.notebook.ghostCellSuggestions.enabled`](positron://settings/positron.assistant.notebook.ghostCellSuggestions.enabled) to `true`. By default, suggestions appear automatically after cell execution. To instead trigger them manually, disable [`positron.assistant.notebook.ghostCellSuggestions.automatic`](positron://settings/positron.assistant.notebook.ghostCellSuggestions.automatic) and click **Get Suggestion** or press {{< kbd mac=Command-Shift-G win=Ctrl-Shift-G linux=Ctrl-Shift-G >}}.

### Follow assistant

When the Assistant is actively editing cells in your notebook, you may want to watch its progress. Select **Follow Assistant** in the notebook editor action bar to have the notebook automatically scroll to cells as they are edited. This lets you review changes in real-time.

> ## Customization & Settings

Author: We have a lot of settings spread across i.e. some are nested under positron, others are not. Adding a list here for discoverability
